barretenberg/cpp/src/barretenberg/relations/generated/avm/sha256.hpp

```diff
@@ -37,7 +37,7 @@ template <typename FF_> class sha256Impl {
         {
             Avm_DECLARE_VIEWS(0);
 
-            auto tmp = ((sha256_sel_sha256_compression * (-sha256_sel_sha256_compression + FF(1))) - FF(0));
+            auto tmp = (sha256_sel_sha256_compression * (-sha256_sel_sha256_compression + FF(1)));
```

Comment:
There must also be some `FF::one()` constant around that we can use instead of constructing `FF(1)`? ;)

Reply:
there is, but this is generated by pil so it would be a special case when encoding a constant